tools/bumper/bumper.go

```diff
@@ -37,8 +38,14 @@ func main() {
 		exitWithError(errors.Wrap(err, "Failed to create github api instance"))
 	}
 
+	bumpingRepo, err := getBumpingRepoComponent(githubApi)
```

Comment:
I actually think making it generic is the preferred practice. 
Not because maybe someone will want to use it in other repo (although who knows)
but because this way I keep the repo specific stuff in specific const variables so it'll be easier to modify stuff later on.

Considering the minimal effort, I think keeping it generic is the preferred way in this case.

Reply:
You can still use constants but don't use generic names for CNAO like BumpingRepo or the like, is really missleading and you know to understand that BumpingRepo is CNAO itself, also CNAO itself is not a network component from CNAO.

Reply:
changed all `BumpingRepo` to `cnaoRepo`
 Done